src/function/scalar/operators/arithmetic.cpp

```diff
@@ -328,6 +334,8 @@ void AddFun::RegisterFunction(BuiltinFunctions &set) {
 
 	functions.AddFunction(GetFunction(LogicalType::TIMESTAMP, LogicalType::INTERVAL));
 	functions.AddFunction(GetFunction(LogicalType::INTERVAL, LogicalType::TIMESTAMP));
+	// we can add lists together
+	functions.AddFunction(GetFunction(LogicalType::LIST(LogicalType::ANY), LogicalType::LIST(LogicalType::ANY)));
```

Comment:
This could just be `functions.AddFunction(ListConcatFun::GetFunction())`, no?

Reply:
Yes, I will change it.